api/environments/dynamodb/types.py

```diff
@@ -69,3 +71,16 @@ def finish_identity_migration(self):
 
     def _save(self):
         return project_metadata_table.put_item(Item=asdict(self))
+
+
+class IdentityOverrideV2(BaseModel):
+    environment_id: str
+    document_key: str
+    environment_api_key: str
+    feature_state: FeatureStateModel
```

Comment:
I might want to integrate this with my code too.

Reply:
I've done this, but I also had to add a new attribute: `identifier: str` to it as well. More detail in a comment on my PR [here](https://github.com/Flagsmith/flagsmith/pull/3116/files/7ce4b10c3fa468ed218b2ab2c706442fd8337ef4#r1421787679).